arkane/kineticsTest.py

```diff
@@ -0,0 +1,94 @@
+# Prof. Richard H. West (r.west@neu.edu) and the RMG Team (rmg_dev@mit.edu)   #
+#                                                                             #
+# Permission is hereby granted, free of charge, to any person obtaining a     #
+# copy of this software and associated documentation files (the 'Software'),  #
+# to deal in the Software without restriction, including without limitation   #
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,    #
+# and/or sell copies of the Software, and to permit persons to whom the       #
+# Software is furnished to do so, subject to the following conditions:        #
+#                                                                             #
+# The above copyright notice and this permission notice shall be included in  #
+# all copies or substantial portions of the Software.                         #
+#                                                                             #
+# THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND, EXPRESS OR  #
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,    #
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE #
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER      #
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING     #
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER         #
+# DEALINGS IN THE SOFTWARE.                                                   #
+#                                                                             #
+###############################################################################
+
+import unittest
+
+from rmgpy.species import TransitionState
+from rmgpy.reaction import Reaction
+
+from arkane.kinetics import KineticsJob
+
+################################################################################
+
+
+class KineticsTest(unittest.TestCase):
+    """
+    Contains unit tests for the Arkane Kinetics module
+    """
+
+    def test_give_tlist_for_kineticsjob(self):
+        """
+        Ensures that the proper temperature ranges are set when Tlist is specified
+        """
+        rxn = Reaction(transitionState=TransitionState())
+        Tlist = [50.7, 100, 300, 800, 1255]
+        kjob = KineticsJob(rxn, Tlist=(Tlist, 'K'))
+        self.assertEqual(min(Tlist), kjob.Tmin.value_si)
+        self.assertEqual(max(Tlist), kjob.Tmax.value_si)
+        self.assertEqual(len(Tlist), kjob.Tcount)
+
+    def test_give_Trange_for_kineticsjob(self):
+        """
+        Ensures that Tlist is set when a range of temperatures is specified
+        """
+        rxn = Reaction(transitionState=TransitionState())
+        kjob = KineticsJob(rxn, Tmin=(50, 'K'), Tmax=(4000, 'K'), Tcount=5)
+        self.assertEqual(5, len(kjob.Tlist.value_si))
+        self.assertAlmostEqual(50, min(kjob.Tlist.value_si))
```

Comment:
why `assertAlmostEqual`?

Reply:
good catch. changed to assertEqual for the Tcount variables